pmm/pmm2-submodules.groovy

```diff
@@ -410,11 +479,14 @@ pipeline {
         always {
             script {
                 if (currentBuild.result == null || currentBuild.result == 'SUCCESS') {
-                    if (env.CHANGE_URL) {
+                    if (env.CHANGE_URL && env.PMM_VER =~ '^2.') {
                         unstash 'IMAGE'
                         def IMAGE = sh(returnStdout: true, script: "cat results/docker/TAG").trim()
                         slackSend channel: '#pmm-ci', color: '#00FF00', message: "[${JOB_NAME}]: build finished - ${IMAGE}"
                     }
+                    else {
+                        addComment("PMM2 Submodules skipped")
```

Comment:
It's better to say what is being built rather than what is skipped :) 
Otherwise it will keep us guessing if v3 is actually triggered.